bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/altair/AltairParams.java

```diff
@@ -244,24 +246,24 @@ static public GuideStarType getGuideStarType(String name, GuideStarType value) {
      */
     public enum Mode implements DisplayableSpType, SequenceableSpType {
```

Comment:
As per our discussion, these aren't exactly new modes in the same sense as these modes, and changing this enum would cause issues, so the list of `AgsStrategy` has been updated instead.